cmd/cli/plugin/package/kctrl/README.md

```diff
@@ -523,6 +549,114 @@ Use "tanzu package repository [command] --help" for more information about a com
     11:47:38PM: packagerepository/standard-repo (packaging.carvel.dev/v1alpha1) namespace: test-ns: DeletionSucceeded
     ```
 
+15. Create a package
+
+    ```sh
+    >>> tanzu package init
+    
+    Welcome! Before we start, do install the latest Carvel suite of tools,
+    specifically ytt, imgpkg, vendir and kbld.
+    
+    Basic Information
+    A package reference name must be at least three '.' separated segments,e.g.
+    samplepackage.corp.com
+    > Enter the package reference name (samplepackage.corp.com): certmanager.carvel.dev
+    
+    Content
+    Please provide the location from where your Kubernetes manifests or Helm chart
+    can be fetched. This will be bundled as a part of the package.
```

Comment:
Both seem to be valid and have the same meaning so I am comfortable with either of them. Changing it would require making a PR in carvel-kapp-controller and then bumping the cli again so I would prefer keeping it the same, but do let me know if you have any strong opinions on this.

Reply:
Yes I meant the sentence ( as ```a``` part of the package) was grammatically wrong. But that is in carvel-kapp-controller and unrelated to this repo